plugins/orca_security/unit_test/test_get_users.py

```diff
@@ -0,0 +1,24 @@
+import sys
+import os
+
+sys.path.append(os.path.abspath("../"))
+
+from unittest import TestCase
+from icon_orca_security.actions.get_users import GetUsers
+from icon_orca_security.actions.get_users.schema import Input
+from unit_test.util import Util
+from unittest.mock import patch
+from parameterized import parameterized
+from insightconnect_plugin_runtime.exceptions import PluginException
+
+
+@patch("requests.request", side_effect=Util.mocked_requests)
+class TestGetUsers(TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.action = Util.default_connector(GetUsers())
+
+    @parameterized.expand(Util.load_parameters("get_users").get("parameters"))
+    def test_get_users(self, mock_request, name, expected):
+        actual = self.action.run()
+        self.assertEqual(actual, expected)
```

Comment:
Same here

Reply:
There is no exceptions for this action that I could test :)